release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -495,6 +507,38 @@ $ operator-sdk new <operator_name> --type=helm \
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874754[*BZ#1874754*])
 
+* If an AWS account is configured to use AWS Organizations service control
+policies (SCPs) that use a global condition to deny all actions or require a
+specific permission, the AWS policy simulator API that validates permissions
+produces a false negative. When the permissions cannot be validated,
+{product-title} AWS installations fail, even if the provided credentials have
+the required permissions for installation.
++
+To work around this issue, you can bypass the AWS policy simulator permissions
+check by setting a value for the `credentialsMode` parameter in the
+`install-config.yaml` configuration file. The value of `credentialsMode` changes
+the behavior of the Cloud Credential Operator (CCO) to one of
+xref:../operators/operator-reference.adoc#cloud-credential-operator_red-hat-operators[three supported modes].
++
+.Example `install-config.yaml` configuration file
++
+[source,yaml]
+----
+apiVersion: v1
+baseDomain: cluster1.example.com
+credentialsMode: Mint <1>
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+...
+----
+<1> This line is added to set the `credentialsMode` parameter to `Mint`.
++
+When bypassing this check, ensure that the credentials you provide have the
+permissions that are required for the specified mode.
```

Comment:
probably should specify exactly what the permissions CCO needs for Mint and Passthrough

Reply:
Meaning the same level of info we had in the 4.5.8 RNs? ([Mint and Passthrough bullets here](https://docs.openshift.com/container-platform/4.5/release_notes/ocp-4-5-release-notes.html#ocp-4-5-8-added-credentialsmode-parameter-for-aws-install-workaround))

If so, that info is going into the CCO entry [here](https://docs.openshift.com/container-platform/4.5/operators/operator-reference.html#cloud-credential-operator_red-hat-operators) (link is to 4.5 version, not expanded as it is for 4.6), so I think maybe linking to that makes sense to encourage reading more about it.

Let me know if you meant something else by "exactly what the permissions CCO needs".

Reply:
I mean that somewhere we should be able to say something like:

For CCO to operate in mint mode (for AWS), it needs these permissions: https://github.com/openshift/cloud-credential-operator/blob/master/pkg/aws/utils.go#L21
For CCO to operate in passthrough mode (for AWS), it needs the combination of all requested permissions for all CredentialsRequests for the version of OpenShift being installed/run (which can be gathered by following the steps for Manual mode).

Reply:
Thank you! Yeah that will def have to be linked, but I'll make sure to explain it and link out to that info in the main docset.